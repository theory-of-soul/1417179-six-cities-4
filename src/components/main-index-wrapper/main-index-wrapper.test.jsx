import React from "react";
import renderer from 'react-test-renderer';
import MainIndexWrapper from "./main-index-wrapper";
import {BrowserRouter} from "react-router-dom";

const children = <div/>;

describe(`MainIndexWrapper component snapshot tests`, () => {
  it(`MainIndexWrapper `, () => {
    const tree = renderer
      .create(
          <BrowserRouter>
            <MainIndexWrapper
              isUserAuth={true}
              userEmail="fake@email.com"
              className="page__main--index-empty"
            >
              {children}
            </MainIndexWrapper>
          </BrowserRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
