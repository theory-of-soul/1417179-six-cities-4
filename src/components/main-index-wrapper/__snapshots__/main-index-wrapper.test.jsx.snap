// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainIndexWrapper component snapshot tests MainIndexWrapper  1`] = `
<div
  className="page page--gray page--main"
>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link header__logo-link--active"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <a
                className="header__nav-link header__nav-link--profile"
                href="/favorites"
                onClick={[Function]}
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                />
                <span
                  className="header__user-name user__name"
                >
                  fake@email.com
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--index"
  >
    <div />
  </main>
</div>
`;
